.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+env:
+  - DJANGO_SETTINGS_MODULE=concordia.settings_test
+python:
+  - "3.6"
+  - "3.6-dev"  # 3.6 development branch
+  - "3.7-dev"  # 3.7 development branch
```

Comment:
I would include 3.7 release as well as the dev branch. We may need to add something like `allowed_failures` for the dev branches, too

Reply:
See https://github.com/travis-ci/travis-ci/issues/9815